pkg/apis/enricher/framework/javascript/nodejs/express_detector.go

```diff
@@ -82,14 +82,30 @@ func GetEnvPort(envPlaceholder string) int {
 	return -1
 }
 
-func getPort(content string, matchIndexes []int) int {
+func GetEnvPortFromDockerfile(envPlaceholder string, path string) int {
```

Comment:
Add doc for `GetEnvPortFromDockerfile`

Reply:
Done in [`503680c` (#23)](https://github.com/devfile/alizer/pull/23/commits/503680cbb78ed8013430282bf6a026fcd1651b45)